### PR TITLE
CB-1567 - env api client with crn

### PR DESCRIPTION
**Pull request by attilapalfi92**

### CODE REVIEW

File: environment/src/main/java/com/sequenceiq/environment/configuration/security/HeaderCrnFilter.java [hunk 1]
```diff
@@ -24,11 +26,17 @@ public HeaderCrnFilter(ThreadLocalUserCrnProvider threadLocalUserCrnProvider) {
     protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain)
             throws ServletException, IOException {
         String userCrn = request.getHeader("x-cdp-actor-crn");
-        if (!Crn.isCrn(userCrn)) {
-            throw new AccessDeniedException("Provided user CRN is invalid!");
+        if (isNotInfoEndpoint(request)) {
```
Comment (attilapalfi92) [inline environment/src/main/java/com/sequenceiq/environment/configuration/security/HeaderCrnFilter.java]: @keyki here is the fix for the info endpoint

File: redbeams-api/src/main/java/com/sequenceiq/redbeams/client/RedbeamsClient.java [hunk 2]
```diff
@@ -1,46 +1,32 @@
...
-        protected ReadbeamsEndpoint(WebTarget webTarget, String crn) {
-            super(webTarget, crn);
-        }
-
-        public DatabaseV4Endpoint databaseEndpoint() {
-            return getEndpoint(DatabaseV4Endpoint.class);
-        }
-    }
-}
+// TODO: please implement this like EnvironmentServiceEndpoints
```
Comment (attilapalfi92) [inline redbeams-api/src/main/java/com/sequenceiq/redbeams/client/RedbeamsClient.java]: we talked about these with @gergopapi2

File: cloud-common/src/main/resources/application.yml [hunk 3]
```diff
@@ -84,9 +84,12 @@ cb:
     capacity.size: 4000
   notification:
     endpoint: http://localhost:3000/notifications
-  freeipa:
+  environment:
```
Comment (attilapalfi92) [inline cloud-common/src/main/resources/application.yml]: this was a mistake form my previous commit

File: environment/src/main/java/com/sequenceiq/environment/configuration/security/HeaderCrnFilter.java (hunk 1 above)
Comment (attilapalfi92) [reply environment/src/main/java/com/sequenceiq/environment/configuration/security/HeaderCrnFilter.java]: @doktoric

File: redbeams-api/src/main/java/com/sequenceiq/redbeams/client/RedbeamsClient.java (hunk 2 above)
Comment (gergopapi2) [reply redbeams-api/src/main/java/com/sequenceiq/redbeams/client/RedbeamsClient.java]: Yes, I filed a jira for that.

**Pull request merged**